build.sh

```diff
@@ -104,12 +104,11 @@ function print_matdbg_help {
 }
 
 # Unless explicitly specified, NDK version will be selected as highest available version within same major release chain
-export FILAMENT_NDK_VERSION=${FILAMENT_NDK_VERSION:-$(cat `dirname $0`/build/android/ndk.version | cut -f 1 -d ".").}
+export FILAMENT_NDK_VERSION=${FILAMENT_NDK_VERSION:-$(cat `dirname $0`/build/android/ndk.version | cut -f 1 -d ".")}
```

Comment:
This shouldn't be exported but passed to CMake as a -D argument like we do for `FILAMENT_ENABLE_JAVA` for instance and various other options.

Reply:
changed to avoid env